frontend/src/components/Metrics/ConfigStep/index.tsx

```diff
@@ -6,7 +6,7 @@ import { useNotificationLayoutEffectInterface } from '@src/hooks/useNotification
 
 const ConfigStep = ({ resetProps }: useNotificationLayoutEffectInterface) => {
   useLayoutEffect(() => {
-    resetProps?.();
+    resetProps();
```

Comment:
React Hook useLayoutEffect has a missing dependency: 'resetProps'. Either include it or remove the dependency array. If 'resetProps' changes too often, find the parent component that defines it and wrap that definition in useCallback

 Check warning on line 10 in frontend/src/components/Metrics/ConfigStep/index.tsx

GitHub Actions
/ frontend-check

React Hook useLayoutEffect has a missing dependency: 'resetProps'. Either include it or remove the dependency array. If 'resetProps' changes too often, find the parent component that defines it and wrap that definition in useCallback





  return (

    <ConfigStepWrapper>

Reply:
done